phpstan.neon

```diff
@@ -27,6 +27,7 @@ parameters:
         -
             message: '#Binary operation "-" between (.)* results in an error#'
             path: 'src/I18n/RelativeTimeFormatter.php'
+        - '#Static property Cake\\Chronos\\(Date|Chronos|MutableDate|MutableDateTime)::\$diffFormatter \(Cake\\Chronos\\DifferenceFormatter\) does not accept Cake\\I18n\\RelativeTimeFormatter#'
```

Comment:
A proper fix for this would be to add an interface for diff formatter to Chronos so that it property can be type hinted with the interface instead of concrete class.